shared/presentation/src/androidUnitTest/kotlin/network/bisq/mobile/presentation/settings/reputation/ReputationScreenUiTest.kt

```
@@ -0,0 +1,90 @@
+
+    @After
+    fun tearDown() {
+        runCatching { stopKoin() }
+    }
+
+    private fun renderScreen() {
+        composeTestRule.setContent {
+            CompositionLocalProvider(
+                LocalUriHandler provides
+                    object : UriHandler {
+                        override fun openUri(uri: String) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)